packages/confirm-dialog/src/styles/vaadin-confirm-dialog-overlay-styles.d.ts

```diff
@@ -5,4 +5,4 @@
  */
 import type { CSSResult } from 'lit';
 
-export const confirmDialogOverlay: CSSResult;
+export declare const confirmDialogOverlayStyles: CSSResult;
```

Comment:
Do we need `declare` here? Other files are using `export const`.

Reply:
Fixed.